tests/gis_tests/tests.py

```diff
@@ -35,7 +43,7 @@ def _get_postgis_func(self, func):
 
 
 @unittest.skipUnless(HAS_POSTGRES, "The psycopg2 driver is needed for these tests")
-class TestPostGISVersionCheck(unittest.TestCase):
+class TestPostGISVersionCheck(TestCase):
```

Comment:
Is there any reason for this change? I would revert it.

Reply:
I changed this to use Django provided test classes. I think the best option for this class is `SimpleTestCase`.
BTW, I reverted it.